app/models/batch_process.rb

```diff
@@ -230,15 +230,22 @@ def create_new_parent_csv
           parent_object = ParentObject.find_or_initialize_by(oid: oid)
         end
 
-        aspace_uri = row['aspace_uri']
-        bib = row['bib']
-        holding = row['holding']
-        item = row['item']
-
-        parent_object.aspace_uri = aspace_uri
-        parent_object.bib = bib
-        parent_object.holding = holding
-        parent_object.item = item
+        parent_object.aspace_uri = row['aspace_uri']
+        parent_object.bib = row['bib']
+        parent_object.holding = row['holding']
+        parent_object.item = row['item']
+
+        if row['aspace_uri'].present? && row['extent_of_digitization'].blank?
```

Comment:
It should check the source to see if the record is using aspace, instead of checking for `row['aspace_uri'].present?`.

check if `metadata_source` or `row['source']` is "aspace"

Reply:
Thank you for the catch.  I've just updated the branch with the suggested change.